patches/chrome-browser-importer-external_process_importer_host.cc.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/chrome/browser/importer/external_process_importer_host.cc b/chrome/browser/importer/external_process_importer_host.cc
+index 9087093838f249ca73cc9fc82d54d5b44e9bb5f2..22e47149317aa6f7a48661d58156f176a2be33e8 100644
+--- a/chrome/browser/importer/external_process_importer_host.cc
++++ b/chrome/browser/importer/external_process_importer_host.cc
+@@ -60,6 +60,7 @@ void ExternalProcessImporterHost::StartImportSettings(
+     return;
+   }
+ 
++  BRAVE_START_IMPORT_SETTINGS
```

Comment:
why are we patching here when we have a subclass?

Reply:
I assume that this was added here because it needs early return in the middle of this method like above upstream condition.

Reply:
```#define CheckForFirefoxLock \
  UnusedMethod() { return false; } \
  virtual bool CheckForFirefoxLock``` and then override in the subclass